ui/html/mentorslist.html

```diff
@@ -7,29 +7,27 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="../css/mentorunique.css">
     <link rel="stylesheet" href="../css/footer.css">
+    <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
 </head>
 
 <body>
     <div class="header">
         <a href="#default" class="logo">FREE MENTORS</a>
         <div class="header-right">
-            <a href="rejected.html">Rejected</a>
-            <a href="accepted.html">Accepted</a>
-            <a href="awaiting.html">Awaiting</a>
+            <a href="./applications.html">Applicatons</a>
             <a class="active" href="mentorslist.html">Browse Mentors</a>
             <a href="../index.html">Logout</a>
         </div>
     </div>
     <div id="id02" class="modal">
             <form class="modal-content animate" action="./mentorslist.html">
                 <div class="imgcontainer">
-                    <h1>create session form</h1>
+                    <h1>Create session form</h1>
                     <p>Please fill in this form to make a session request to this mentor.</p>
                     <span onclick="document.getElementById('id02').style.display='none'" class="close"
                         title="Close Modal">×</span>
                 </div>
                 <div class="container">
-                    <label for="remark"><b>questions</b></label>
```

Comment:
i deleted this label also cause i specifiy also in placeholder of textarea that user should enter their questions there